csharp/Managed/Events/SymbolEventArgs.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using LibPDBinding.Managed.Data;
+
+namespace LibPDBinding.Managed.Events
+{
+	public class SymbolEventArgs : EventArgs
+	{
+		public string Receiver { get; private set; }
+
+		public Symbol Symbol { get; private set; }
```

Comment:
This could also contain a C# `string` instead of the `IAtom` encapsulation, see comment to ListEventArgs.cs